azure-ts-comosdb-with-apiconnection/README.md

```diff
@@ -0,0 +1,125 @@
+[![Deploy](https://get.pulumi.com/new/button.svg)](https://app.pulumi.com/new)
+
+# Deploy an Azure CosmosDB container and an API Connection
+
+At the time of this writting, there was no API support from either Terraform or Azure (or both)
+to handle CosmosDB contaiers and API Connections. Therefore, this is a how-to:
+ - Levarage Azure's CosmosDB SDK to create a CosmosDB container
+ - Use an ARM template to create an API Connection
+
+## Prerequisites
+
+- [Node.js](https://nodejs.org/en/download/)
+- [Download and install the Pulumi CLI](https://www.pulumi.com/docs/reference/install/)
+- [Connect Pulumi with your Azure account](https://www.pulumi.com/docs/reference/clouds/azure/setup/) (if your `az` CLI is
+      configured, this will just work)
+
+## Running the App
+
+1. Create a new stack:
+
+    ```sh
+    $ pulumi stack init dev
+    ```
+
+2. Set the required configuration variables for this program, and log into Azure:
+
+    ```bash
+    $ pulumi config set azure:environment public
+    $ az login
+    ```
+
+3. Perform the deployment:
+
+    ```sh
+    $ pulumi up
+    Updating (dev):
+
+        Type                               Name                                   Status      Info
+        pulumi:pulumi:Stack                azure-cosmosdb-wtih-apiconnection-dev              3 messages
+    +   ├─ azure:core:ResourceGroup        dev-rg                                 created
+    +   ├─ azure:storage:Account           devsa                                  created
+    +   ├─ azure:cosmosdb:Account          dev-db-acc                             created
+    +   ├─ azure:cosmosdb:SqlDatabase      dev-db                                 created
+    +   ├─ pulumi-nodejs:dynamic:Resource  dev-db-container                       created
+    +   └─ azure:core:TemplateDeployment   db-connection                          created
+
+    Resources:
+        + 6 created
+        1 unchanged
+
+    Duration: 29m15s
+    ```
+
+## Next Steps
+As an example, can use the API Connection to establish a Logic App - ComosDB relation:
```

Comment:
Problem is that this part gets a bit tricky because of the requirement of the ARM template in order to support the connection parameters... and if I add it without any context, might create more confusion.

Instead what I am going to do, is writing a few posts about the whole case, one which reflects the need of using this approach in order to be able to utilise the API Connection.

What do you think?

Reply:
Do you have that full example so that I could take a look? Maybe we can find that balance.
And the blog posts are certainly very welcome!